app/templates/index.hbs

```diff
@@ -0,0 +1,7 @@
+{{#each model as |reminder|}}
```

Comment:
This should live in the `reminders` template since it displays all reminders. We actually don't need this file at all.